design-library/src/components/BccProgress/BccProgress.css

```diff
@@ -14,17 +14,21 @@
 
     /* Size */
     .bcc-progress-container .bcc-form-label-lg .bcc-form-label-optional {
-        @apply text-label;
+        @apply text-label-base;
```

Comment:
Additional comment here: In the [Figma file](https://www.figma.com/file/6GvZltnPMH3yynzf7287Iu/BCC-Components---Master?type=design&node-id=377-10336&mode=design&t=WfLbCRi9JCILf82s-0) it's possible to see the actual typography styles, see below. 

![image](https://github.com/bcc-code/bcc-design/assets/39056299/39d2df75-2ffe-4586-81b4-c7451cf1dc2d)

Here you can see that the actual text styles are `heading xs` and `body sm`. I think we should keep our implementation as close as possible to the design, so I suggest changing those, perhaps by no longer using the form label component but implementing it directly.